### ISSUE TITLE
Hangs with MacOS 13 Ventura

### ISSUE THREAD

**Issue by frakman1** [osxfuse/osxfuse#901]
I installed the latest Macfuse: `macfuse-4.4.0.pkg` and sshfs: `sshfs-2.5.0.pkg`
I am using a MacBook Pro M1 Max with MacOS 13.0

When mounting a remote folder, it hangs and occasionally prints out some error logs like:
`   unique: 10, error: -2 (No such file or directory), outsize: 16`

Even though I am mounting a remote folder with just 3 files, the sshfs logs indicate looking for files in the root or elsewhere (`.Spotlight-V100`, `.metadata_never_index` etc)

Full log:

```bash
$sudo sshfs -d frak@frakbookair:/Users/frak/Desktop /Users/frak/sshfs_mounts/frakbookair/ -o IdentityFile=/Users/frak/.ssh/id_rsa
FUSE library version: 2.9.9
nullpath_ok: 0
nopath: 0
utime_omit_ok: 0
unique: 2, opcode: INIT (26), nodeid: 0, insize: 56, pid: 4279
INIT: 7.19
flags=0xef000008
max_readahead=0x00100000
   INIT: 7.19
   flags=0x00000010
   max_readahead=0x00100000
   max_write=0x02000000
   max_background=0
   congestion_threshold=0
   unique: 2, success, outsize: 40
unique: 2, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 0
statfs /
unique: 3, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 4279
statfs /
   unique: 3, success, outsize: 96
   unique: 2, success, outsize: 96
unique: 2, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 119
statfs /
unique: 3, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 171
statfs /
   unique: 3, success, outsize: 96
   unique: 2, success, outsize: 96
unique: 3, opcode: ACCESS (34), nodeid: 1, insize: 48, pid: 782
access / 00
   unique: 3, error: -78 (Function not implemented), outsize: 16
unique: 2, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 171
statfs /
unique: 3, opcode: GETATTR (3), nodeid: 1, insize: 56, pid: 779
getattr /
   unique: 2, success, outsize: 96
unique: 2, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 171
statfs /
   unique: 3, success, outsize: 136
   unique: 2, success, outsize: 96
unique: 3, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 1455
statfs /
unique: 2, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 748
statfs /
unique: 4, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 855
statfs /
unique: 5, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 872
statfs /
unique: 6, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 715
statfs /
unique: 7, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 779
statfs /
unique: 8, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 116
statfs /
unique: 9, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 1399
statfs /
unique: 10, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 903
statfs /
unique: 11, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 1399
statfs /
   unique: 2, success, outsize: 96
   unique: 3, success, outsize: 96
   unique: 4, success, outsize: 96
   unique: 5, success, outsize: 96
   unique: 6, success, outsize: 96
   unique: 11, success, outsize: 96
   unique: 8, success, outsize: 96
   unique: 9, success, outsize: 96
   unique: 10, success, outsize: 96
   unique: 7, success, outsize: 96
unique: 2, opcode: GETXATTR (22), nodeid: 1, insize: 77, pid: 779
unique: 3, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 0
statfs /
getxattr / com.apple.FinderInfo 32 0
   unique: 2, success, outsize: 48
   unique: 3, success, outsize: 96
unique: 4, opcode: OPENDIR (27), nodeid: 1, insize: 48, pid: 116
opendir flags: 0x0 /
   opendir[0] flags: 0x0 /
   unique: 4, success, outsize: 32
unique: 5, opcode: RELEASEDIR (29), nodeid: 1, insize: 64, pid: 116
releasedir[0] flags: 0x0
   unique: 5, success, outsize: 16
unique: 6, opcode: GETATTR (3), nodeid: 1, insize: 56, pid: 116
getattr /
   unique: 6, success, outsize: 136
unique: 7, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 116
statfs /
   unique: 7, success, outsize: 96
unique: 8, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 116
statfs /
   unique: 8, success, outsize: 96
unique: 11, opcode: OPENDIR (27), nodeid: 1, insize: 48, pid: 116
opendir flags: 0x0 /
   opendir[0] flags: 0x0 /
   unique: 11, success, outsize: 32
unique: 10, opcode: STATFS (17), nodeid: 1, insize: 40, pid: 116
statfs /
   unique: 10, success, outsize: 96
unique: 2, opcode: LOOKUP (1), nodeid: 1, insize: 45, pid: 116
LOOKUP /DCIM
getattr /DCIM
   unique: 2, error: -2 (No such file or directory), outsize: 16
unique: 9, opcode: LOOKUP (1), nodeid: 1, insize: 76, pid: 116
LOOKUP /.metadata_never_index_unless_rootfs
getattr /.metadata_never_index_unless_rootfs
   unique: 9, error: -2 (No such file or directory), outsize: 16
unique: 3, opcode: LOOKUP (1), nodeid: 1, insize: 62, pid: 116
LOOKUP /.metadata_never_index
getattr /.metadata_never_index
   unique: 3, error: -2 (No such file or directory), outsize: 16
unique: 4, opcode: LOOKUP (1), nodeid: 1, insize: 68, pid: 116
LOOKUP /.metadata_direct_scope_only
getattr /.metadata_direct_scope_only
   unique: 4, error: -2 (No such file or directory), outsize: 16
unique: 5, opcode: LOOKUP (1), nodeid: 1, insize: 56, pid: 116
LOOKUP /.Spotlight-V100
getattr /.Spotlight-V100
   unique: 5, error: -2 (No such file or directory), outsize: 16
unique: 6, opcode: GETATTR (3), nodeid: 1, insize: 56, pid: 116
getattr /
   unique: 6, success, outsize: 136

```

The **remote** folder only contains these files:
```bash
$ls -lasth /Users/frak/Desktop
total 152
  0 drwxr-xr-x+ 128 frak  staff   4.0K Jul  5 13:22 ..
 16 -rw-r--r--@   1 frak  staff   6.0K Dec 23  2021 .DS_Store
  0 drwx------+   5 frak  staff   160B Dec 23  2021 .
136 -rw-rw-rw-@   1 frak  staff    71B Jun 21  2017 lifx.webloc
  0 -rw-r--r--    1 frak  staff     0B Nov 23  2013 .localized
```

**Comment by bfleischer** [comment]
Could you check if the issue still exists with macFUSE 4.4.1? The 4.4.1 update addresses a bug that might be to blame for the hang you are seeing.

**Comment by frakman1** [comment]
Thank you for the update.
After I ran into this problem with `osxfuse`, I found an alternate solution using `macSSHS`/`macfuse` :

https://github.com/tormodwill/macSSHFS

In conjuction with `brew install macfuse`, I was able to get `sshfs` to work that way.

Unfortunately, since they have different dependencies (`load_macfuse` vs `load_osxfuse`), I won't be able to test the `osxfuse` method as I believe I had uninstalled it and I don't think they can both coexist.

For reference, [this](https://github.com/tormodwill/macSSHFS/discussions/4) was the discussion whereby I got `sshfs` to finally work after discovering the remnants of the original `osxfuse` installation.
